src/core/thread/mle_router.hpp

```diff
@@ -565,7 +581,6 @@ class MleRouter : public Mle
 
     static constexpr uint8_t kLinkRequestMinMargin    = OPENTHREAD_CONFIG_MLE_LINK_REQUEST_MARGIN_MIN;
     static constexpr uint8_t kPartitionMergeMinMargin = OPENTHREAD_CONFIG_MLE_PARTITION_MERGE_MARGIN_MIN;
-    static constexpr uint8_t kChildRouterLinks        = OPENTHREAD_CONFIG_MLE_CHILD_ROUTER_LINKS;
```

Comment:
Suggest keeping this constant so we can use it when we initialize `mChildRouterLinks` in the constructor.

Reply:
done.